src/main/scala/com/atomist/rug/runtime/js/JavaScriptContext.scala

```diff
@@ -78,13 +84,16 @@ class JavaScriptContext(val rugAs: ArtifactSource,
   // Require all the Atomist stuff
   atomistContent
     .filter(_ => true, atomistConfig.isJsSource)
-    .allFiles.foreach(evaluate)
+    .allFiles
+    .foreach(evaluate)
+
 
   /**
     * Evaluate the given file, with proper exports handling.
     * NB: Must be a file from atomistContent.
     */
   def evaluate(f: FileArtifact): Unit = {
+    require(!f.content.contains("nodeName()"))
```

Comment:
These checks are _super_ expensive. We already have one for removing some source maps from the code, and that is super slow too....